src/custom/api/gnosisProtocol/api.ts

```diff
@@ -279,7 +279,18 @@ async function _handleQuoteResponse<T = any, P extends QuoteQuery = QuoteQuery>(
   params?: P
 ): Promise<T> {
   if (!response.ok) {
-    const errorObj: ApiErrorObject = await response.json()
+    const errorObj: ApiErrorObject = await response.json().catch((error) => {
```

Comment:
We needed to handle this error. Now it creates a new sentry error called `HandleQuoteResponseJsonParse` and sends the response status